src/components/views/settings/DevicesPanel.js

```diff
@@ -209,24 +221,27 @@ export default class DevicesPanel extends React.Component {
 
         devices.sort(this._deviceCompare);
 
-        const deleteButton = this.state.deleting ?
+        const signOutButton = this.state.deleting?
             <Spinner w={22} h={22} /> :
-            <AccessibleButton onClick={this._onDeleteClick} kind="danger_sm">
-               { _t("Delete %(count)s sessions", {count: this.state.selectedDevices.length}) }
+            <AccessibleButton onClick={this._onSignOutClick} kind="danger"
+                disabled={this.state.selectedDevices.length === 0}>
```

Comment:
when we break these out we break them all out, with the `>` being on its own line.

Reply:
Fixed